index.ts

```diff
@@ -100,13 +101,14 @@ export const Task: resolvers.Task = {
   parentTask(_, { obj }) {
     return TaskCollection.one!({ id: obj.parentTaskId }, {} as any);
   },
-  async postMessage({ type, message }, { self }) {
-    if (type !== "comment" && type !== "description") {
-      throw new Error("Invalid message type");
-    }
+  async comment({ message }, { self }) {
     const { id: taskId } = self.$argsAt(root.tasks.one)!;
-    // FIXME: fails saying `taskId` is not a UUID???
-    return await api.post(`activities`, { type, message, taskId });
+    const { id } = await api.post<{ id: string }>(`activities`, {
+      type: "comment",
+      message,
+      taskId,
+    });
+    return root.activities.one({ id });
   },
```

Comment:
Few changes here:
1. I just made the `comment` action explicit. We could have an `updateDescription` action potentially as well, but given that we can't represent enums (yet) for fields this seemed like the more ergonomic API. 
2. Instead of just returning `Activity` directly the action now returns `Gref<Activity>`. @juancampa and I had talked about some different options here and things we could potentially do in the future to improve this... I don't think this is really the _best_ experience, but it's sufficient.